test/e2e/before-after/mixed/tests/01-nested.test.brs

```diff
@@ -31,6 +31,16 @@ function main(args as object) as object
             m.assert.equal(m.counterB, 11, "counterB must be incremented for each test case")
         end sub)
 
+        m.it("check _brs_.testData", sub()
```

Comment:
I love the idea of more tests, but since `_brs_.testData` is now entirely controlled by the `brs` repo (and is tested there), I don't think we should add test cases here for it.

Reply:
done, I've removed UT